actions.yml

```diff
@@ -66,6 +66,10 @@ BobAnkh/auto-generate-changelog:
   '*':
     expires_at: 2025-08-01
     keep: true
+dorny/test-reporter:
+  '*':
```

Comment:
We don't allow new wildcards in the allowlist, could you please use the hash?

Reply:
added hash, expiry date and tag. Hope that is what you had in mind.